platforms/minigame/common/engine/cache-manager.js

```diff
@@ -183,10 +183,11 @@ const cacheManager = {
             caches.push({ originUrl: key, url: val.url, lastTime: val.lastTime });
         });
         caches.sort((a, b) => a.lastTime - b.lastTime);
-        caches.length = Math.floor(caches.length / 3);
-        if (caches.length === 0) {
-            cleaning = false;
-            return;
+        // cache length above 3 then clear 1/3, or clear all caches
+        if (caches.length < 3) {
+            console.warn('Due to caching large files in the game, there is insufficient storage space. Now starting forced cleaning.');
```

Comment:
This warning message takes **106** bytes. Could it be shorter? For instance:

> Insufficent storage, cleaning

Reply:
ok，i will try